ElectronClient/app/gui/SideBar.jsx

```diff
@@ -267,14 +267,18 @@ class SideBarComponent extends React.Component {
 		if (!itemId || !itemType) throw new Error('No data on element');
 
 		let deleteMessage = '';
+		let buttonLabel = '';
```

Comment:
Could you give it `_('Remove')` as a default value please? Then you can remove the two `buttonLabel = _('Remove');` below. That way, if there's ever another condition added below, the button label would still have a good default.

Reply:
Sure, will do.